code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -197,7 +197,7 @@
 	item_state = "pepperspray"
 	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
-	volume = 40
+	volume = 50
 	stream_range = 4
 	amount_per_transfer_from_this = 5
 	list_reagents = list(/datum/reagent/consumable/condensedcapsaicin = 40)

```

Comment:
This also needs changing if you want to increase the volume unless you want to force people to refill their initial pepperspray for whatever reason

Reply:
ah thanks nemvar!